winml/lib/Api/LearningModelBinding.cpp

```diff
@@ -212,101 +219,160 @@ bool LearningModelBinding::HasKey(hstring const& key) {
 void LearningModelBinding::Split(
     Windows::Foundation::Collections::IMapView<hstring, Windows::Foundation::IInspectable>& first,
     Windows::Foundation::Collections::IMapView<hstring, Windows::Foundation::IInspectable>& second) {
+  ORT_UNUSED_PARAMETER(first);
+  ORT_UNUSED_PARAMETER(second);
```

Comment:
what is this method for? looks like all it does is not use parameters and then throw.

Reply:
it's a method from IMapView.  [https://docs.microsoft.com/en-us/uwp/api/windows.foundation.collections.imapview-2.split](https://docs.microsoft.com/en-us/uwp/api/windows.foundation.collections.imapview-2.split).   we don't support it.

---
In reply to: [354044697](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354044697) [](ancestors = 354044697)

Reply:
interesting.  reading the API docs.   it says we should not through, but instead just set them both to nullptr.  ima do that. great catch !

---
In reply to: [354618352](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354618352) [](ancestors = 354618352,354044697)

Reply:
FYI - this was all in market code.  just moved over.

---
In reply to: [354618577](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354618577) [](ancestors = 354618577,354618352,354044697)